### PR TITLE
Update dependabot.yml for release-1.28

### PR DESCRIPTION
**Pull request by MartinForReal**
#### What type of PR is this?
/kind testing

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -249,6 +249,55 @@ updates:
...
+      time: "01:00"
+      timezone: "Asia/Shanghai"
+    labels:
+      - "area/dependency"
+      - "release-note-none"
+      - "ok-to-test"
+      - "kind/cleanup"
+      - "lgtm"
+      - "approved"
+      - "release-1.27"
```
Comment (feiskyer) [inline .github/dependabot.yml:266]: release-1.28

Comment (jwtty) [inline .github/dependabot.yml:266]: release-1.28

Comment (MartinForReal) [reply .github/dependabot.yml]: updated. Thanks!

Comment (MartinForReal) [reply .github/dependabot.yml]: updated. Thanks!

**Pull request merged**